File: tasks/prac6_python/task6.py
import random

secret_number = random.randint(1, 100)
attempts = 0

print("Загадано число от 1 до 100. Попробуйте угадать!")

while True:
    guess = int(input("Ваша попытка: "))
    attempts += 1

    if guess < secret_number:
        print("Больше!")
    elif guess > secret_number:
        print("Меньше!")
    else:
        print(f"Поздравляем! Вы угадали число {secret_number} за {attempts} попыток.")
        break

File: tasks/prac6_js/task6.js
const prompt = require('prompt-sync')();

const secretNumber = Math.floor(Math.random() * 100) + 1;
let attempts = 0;

console.log("Загадано число от 1 до 100. Попробуйте угадать!");

while (true) {
    const guess = parseInt(prompt("Ваша попытка: "));
    attempts++;
    
    if (guess < secretNumber) {
        console.log("Больше!");
    } else if (guess > secretNumber) {
        console.log("Меньше!");
    } else {
        console.log(`Поздравляем! Вы угадали число ${secretNumber} за ${attempts} попыток.`);
        break;
    }
}